concent_api/core/transfer_operations.py

```diff
@@ -84,7 +83,7 @@ def store_pending_message(
     receive_queue.save()
     if payment_message is not None:
         payment_committed_message = PaymentInfo(
-            payment_ts                  = datetime.datetime.fromtimestamp(payment_message.payment_ts, timezone.utc),
+            payment_ts                  = parse_timestamp_to_utc_datetime(payment_message.payment_ts),
```

Comment:
please format this block according to our coding style ;)

Reply:
:+1: done